scripts/git_env.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Name::        git_env.sh
+# Description:: Use this script to configure local git env,
+#               including commit message validation
+# Author::      Darren Murray (<dmurray-lacework@lacework.net>)
+#
+
+readonly commit_hook=.git/hooks/prepare-commit-msg
+readonly new_commit_hook=scripts/githooks/prepare-commit-msg
+readonly project_name=go-sdk
+
+create_prepare_commit_msg(){
+  if [ ! -f $commit_hook ]; then
+    log "Adding git commit hooks"
+    cp scripts/githooks/prepare-commit-msg $commit_hook
+    chmod +x $commit_hook
+  else 
+    log "Git commit hook already exists"
+    update
+  fi
+}
+
+update() {
```

Comment:
With a link we wouldn't have to worry about updating. 😏

Reply:
Cool, good suggestion Salim. I'll do that instead to save the updating trouble.